2022-08-what-is-dca.md

```diff
@@ -0,0 +1,44 @@
+---
+title: DCA one of the most profitable strategy
+subtitle: Dollar cost averaging is one of the most profitable strategy for a long term investment
+slug: what-is-dca
+tags: cryptocurrency, trading, strategy, how-to, blog
+cover: https://raw.githubusercontent.com/Drakkar-Software/OctoBot/assets/OctoBot-icon-only.png
+domain: blog.octobot.online
+--- 
+
+# Why dollar cost averaging is one of the most profitable strategy for a long term investment
+
+When it comes to investing your money, there are a lot of different strategies that you can choose from. But if you're looking for a strategy that is reliable and profitable in the long term, dollar cost averaging is definitely one to consider. In this article, we'll explain exactly what dollar cost averaging is and why it's such a great strategy for long-term investors!
+
+# What is dollar cost averaging strategy
+
+Dollar cost averaging is an investing technique whereby an investor purchases a fixed dollar amount of a particular investment on a regular schedule, regardless of the cryptocurrency price. By buying the same dollar amount each time, more cryptocurrency are purchased when prices are low and fewer cryptocurrency are bought when prices are high. Over time, this technique can help reduce the effects of volatility and risk on an investment portfolio.
+
+One of the main advantages of dollar cost averaging is that it takes the emotion out of investing. By buying into an investment on a regular schedule, investors can avoid the temptation to "time the market" by trying to predict when prices will go up or down. This strategy can also help investors stick to their long-term goals, even when markets are going through short-term ups and downs.
+
+# Why dollar cost averaging is even more interesting in the cryptocurrencies world
+
+As we all know, cryptocurrencies are a volatile market. The prices of Bitcoin, Ethereum, Litecoin and other altcoins can fluctuate wildly from day to day, and even from hour to hour. This makes investing in cryptocurrencies a risky proposition.
+
+However, there is a way to mitigate this risk somewhat, and that is by dollar cost averaging (DCA). DCA is an investing strategy where you spread your investment into multiple smaller investments over time, rather than investing all at once.
+
+For example, let's say you want to invest $1,000 in Bitcoin. You could do this by buying 10 Bitcoins all at once at the current market price. However, if the price of Bitcoin falls tomorrow, you will have lost 10% of your investment.
```

Comment:
Maybe we should change the numbers here to be closer to the current price and be more relatable